tests/lcs/core/array/combine.livecodescript

```diff
@@ -35,7 +35,12 @@ on TestCombineStrings
    sort lines of tArray by item 1 of each
    sort lines of tUncombinedData by item 1 of each
    
-   TestAssert "Combine strings", tUncombinedData is tArray
+   if the platform is "win32" then
```

Comment:
I'm not clear why these are broken on win32 and not on other platforms - unless the input data is being generated differently... My concern is that the test is actually broken on all platforms, and it is just serendipitous that it is okay on Linux/macOS...

Reply:
As discussed offline, I'm inclined to mark this test as broken for all platforms as.